tools/docker/centos_8.dockerfile

```diff
@@ -0,0 +1,146 @@
+FROM centos:latest
+
+ARG PREFIX=/usr/local
+
+# Support multiarch
+RUN rpm --target i386
+
+RUN cd /etc/yum.repos.d/
+RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
+RUN sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
+
+# Add rocm repository
+RUN sh -c 'echo -e "[rocm]\nname=rocm\nbaseurl=http://repo.radeon.com/rocm/centos8/5.2.5/main/\nenabled=1\ngpgcheck=1\ngpgkey=http://repo.radeon.com/rocm/rocm.gpg.key" > /etc/yum.repos.d/rocm.repo'
+
+RUN yum clean all && \
+    yum install -y wget \
+    dnf-plugins-core \
+    clang \
+    epel-release
+
+#For CentOS 8.3/8.4
+RUN sh -c 'echo yum config-manager --set-enabled PowerTools'
+
+# Install dependencies
+RUN dnf update  -y && dnf makecache --refresh
+RUN dnf groupinstall -y "Development Tools" --setopt=group_package_types=mandatory,default,optional
+
+RUN dnf --enablerepo=powertools install -y --nobest \
+    yum-utils \
+    #build-essential \
+    #clang-format-10 \
```

Comment:
There are a number of these lines commented out.  Just remove them

Reply:
will do

Reply:
Done